src/simulation_strategies/UniformLossSimulationStrategy.py

```diff
@@ -8,23 +8,23 @@
 
 from src.BosonSamplingSimulator import BosonSamplingSimulator
 from src.simulation_strategies.FixedLossSimulationStrategy import FixedLossSimulationStrategy
-from src.simulation_strategies.SimulationStrategy import SimulationStrategy
+from src.simulation_strategies.SimulationStrategyInterface import SimulationStrategyInterface
 
 
-class UniformLossSimulationStrategy(SimulationStrategy):
+class UniformLossSimulationStrategyInterface(SimulationStrategyInterface):
```

Comment:
Nitpick: Why is this class called an `interface`? What does it interface to?

Reply:
It obviously is not. For some reason, a lot of classes got renamed like that after I renamed SimulationStrategy (or so I think). I corrected that.